### ISSUE TITLE
Failed to resolve hostname device: Name or service not known

### ISSUE THREAD

**Issue by dhreben** [zebrunner/mcloud-agent#324]
**Appium log:** 

```

[info] [CaptureArtifacts] trying to send 'on': nc device 2223
nc: getaddrinfo for host "device" port 2223: Name or service not known
[info] [CaptureArtifacts] generating video file 41c144d1-8937-4ca4-b0e8-9dd5c951344b.mp4...
[MCLOUD] .recording-artifact-41c144d1-8937-4ca4-b0e8-9dd5c951344b file created successfully.

```

**Comment by azarouski** [comment]
fixed #325

**Comment by dhreben** [comment]
Fixed

```
[info] [CaptureArtifacts] trying to send 'on': nc device 2223
Connection to device 2223 port [tcp/*] succeeded!
```
